locales/fr.json

```diff
@@ -101,6 +103,7 @@
   "student_debt.1": "Oui.",
   "student_debt.2": "Non.",
   "student_debt.title": "Dette d’études",
+  "student_financial_aid.header": "Plus d’aide financière aux études pour l’année scolaire 2010-21",
```

Comment:
Should be 2020-21 see #376

Reply:
Addressed this in the recent round of changes--thanks for spotting!